docs/integrations/containers-orchestration/opentelemetry/vmware-opentelemetry.md

```diff
@@ -0,0 +1,254 @@
+
+The dashboards provide insight into key metrics such as CPU, memory, disk utilization at different infrastructure level like vm, host, datacenter, resource pool and clusters. This enables you to determine capacity constraints and troubleshoot operational issues related to over-provisioning, changes to configuration, and VM movement.
+
+VMWare metrics are collected through the [vCenter Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/vcenterreceiver) of OpenTelemetry.
+
+See the [vSphere product page](https://www.vmware.com/products/vsphere.html) for more information on VMware hybrid cloud.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/VMWare-OpenTelemetry/VMWare-Schematics.png' alt="Schematics" />
+
+## Prerequisites
+VMWare metrics are collected through the [vCenter Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/vcenterreceiver) of OpenTelemetry.
+
+This receiver has been built to support ESXi and vCenter versions:
+
+- 8
+- 7.0
+
+A “Read Only” user assigned to a vSphere with permissions to the vCenter server, cluster and all subsequent resources being monitored must be specified in order for the receiver to retrieve information about them.
+
+## Collection configuration and app installation
+
+import ConfigAppInstall from '../../../reuse/apps/opentelemetry/config-app-install.md';
+
+<ConfigAppInstall/>
+
+### Step 1: Set up Collector
+
+import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
+
+<SetupColl/>
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/VMWare-OpenTelemetry/VMWare-Collector.png' style={{border:'1px solid gray'}} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step, we will be configuring the yaml file required for VMWare Collection. Here are the list of parameters which are required:
+
+- **Endpoint** to the vCenter Server or ESXi host that has the sdk path enabled. This is a required parameter. The expected format is `<protocol>://<hostname>`
+
+i.e: `https://vcsa.hostname.localnet`
+- **Username** of User which has access to vcenter server
+- **Password** of User which has access to vcenter server
+- **collection_interval** this receiver collects metrics on an interval. If the vCenter is fairly large, this value may need to be increased. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h` 
+
+You can add any custom fields which you want to tag along with the data ingested in sumo. Click on the **Download YAML File** button to get the yaml file.
+
+For Linux platform, click **Download Environment Variables File** to get the file with the password which is supposed to be set as environment variable.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/VMWare-OpenTelemetry/VMWare-YAML.png' style={{border:'1px solid gray'}} alt="YAML" />
```

Comment:
This image does not load in my local build. When I go to the URL, it says "AccessDenied".

Reply:
Uploaded to S3 and made public. No changes required in the PR.